mk/mk.deps

```diff
@@ -218,12 +218,15 @@ src/oldreader_api.o: src/oldreader_api.cpp include/oldreader_api.h \
  include/remote_api.h include/configcontainer.h include/configparser.h \
  include/urlreader.h include/cache.h include/rss.h include/matcher.h \
  filter/FilterParser.h include/utils.h include/logger.h config.h \
- include/strprintf.h
+ include/strprintf.h include/password.h
 src/oldreader_urlreader.o: src/oldreader_urlreader.cpp \
  include/oldreader_api.h include/remote_api.h include/configcontainer.h \
  include/configparser.h include/urlreader.h include/cache.h include/rss.h \
  include/matcher.h filter/FilterParser.h include/utils.h include/logger.h \
  config.h include/strprintf.h
+src/password.o: src/password.cpp include/password.h include/utils.h \
+ include/logger.h config.h include/strprintf.h include/configcontainer.h \
+ include/configparser.h
```

Comment:
Did `make depslist` include `configcontainer` and `configparser` here? That's a bit weird.